### DISCUSSION TITLE
When does Al Dente take effect after cold starting the MacBook?

### DISCUSSION THREAD

**Discussion by 1c383r9** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1209] [Q&A] [answered]
My guess it's after the apple logo screen? Or after logging in? Hope someone reveals this.

Appreciate it!

**Comment by MatthiasKerbl** [comment]
> My guess it's after the apple logo screen? Or after logging in? Hope someone reveals this.
> 
> Appreciate it!

Hi @2ndMessiah,

Depending on your MacBook model and your setup, AlDente will start and take effect within the first minute after starting your MacBook.

Best,
Matthias

**Reply by 1c383r9** [reply]
Thx for the quick reply !! 
Mine is a M1 Pro model. How about that ? @MatthiasKerbl

**Reply by MatthiasKerbl** [reply, marked as answer]
Hi @2ndMessiah,

For your model, it should take effect relatively quickly after login, depending on how many and which applications start during restart.

Best,
Matthias

**Comment by 1c383r9** [comment]
Got it.  I still have to login to effectuate it. That answered my question.
Appreciate it !

Matthias Kerbl ***@***.***>于2024年4月15日 周一22:32写道：
